modules/pages/client/views/navigation.client.view.html

```diff
@@ -31,7 +30,6 @@ <h4 class="list-group-item-heading" ng-bind="::app.user.displayName"></h4>
      ui-sref="invite"
      class="list-group-item">
     Invite Friends
-    <span class="label label-primary">New!</span>
   </a>
```

Comment:
While at it, should we remove this whole "invite friends" link since invitations are now disabled anyway?

Reply:
good call!